data-prepper-plugins/sqs-source/src/main/java/org/opensearch/dataprepper/plugins/source/sqs/SqsService.java

```diff
@@ -80,9 +85,11 @@ public void start() {
                 final PluginModel codecConfiguration = queueConfig.getCodec();
                 final PluginSetting codecPluginSettings = new PluginSetting(codecConfiguration.getPluginName(), codecConfiguration.getPluginSettings());
                 final InputCodec codec = pluginFactory.loadPlugin(InputCodec.class, codecPluginSettings);
-                sqsEventProcessor = new SqsEventProcessor(new BulkSqsMessageHandler(codec));
+                MessageFieldStrategy bulkStrategy = new JsonBulkMessageFieldStrategy(codec);
+                sqsEventProcessor = new SqsEventProcessor(new RawSqsMessageHandler(bulkStrategy));
             } else {
-                sqsEventProcessor = new SqsEventProcessor(new RawSqsMessageHandler());
+                MessageFieldStrategy standardStrategy = new StandardMessageFieldStrategy();
+                sqsEventProcessor = new SqsEventProcessor(new RawSqsMessageHandler(standardStrategy));
```

Comment:
This line and line 89 are the same. Refactor to use the same line. Just get the `MessageFieldStrategy` from the conditional.

Reply:
that makes sense, i made the change